app/models/paperclip_stub.rb

```diff
@@ -1,22 +0,0 @@
-module Paperclip
-  def run cmd, params = "", expected_outcodes = 0
-    case cmd
-    when "identify"
-      return "100x100"
-    when "convert"
-      return
```

Comment:
This was blocking `convert` execution in production. This stub still exists for rspec tests.